### PR TITLE
Upcoming RAML site onetrust implementation

### PR DESCRIPTION
**Pull request by ulisesbuena**
For the upcoming version of the RAML website cookies preferences link was added as well as proper styling. GTM tag was added to the head tag, analytics was removed since now it will be added trough GTM.

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -69,14 +69,15 @@
 
 
 // GDPR script
+/*
```
Comment (jstoiko) [inline js/script.js]: Let's delete this section if it's not used anymore

**Review by jstoiko — CHANGES_REQUESTED** [review]
One minor comment otherwise looks good. Thanks!

File: js/script.js (hunk 1 above)
Comment (ulisesbuena) [reply js/script.js]: I just did!

**Comment by ulisesbuena** [comment]
The issues were addressed.

File: _includes/header.html [hunk 2]
```diff
@@ -33,6 +33,14 @@
...
         });
     </script>
+    <!-- Google Tag Manager -->
+    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+    })(window,document,'script','dataLayer','GTM-TKR6L22');</script>
+    <!-- End Google Tag Manager -->
+    <!--
```
Comment (jstoiko) [inline _includes/header.html]: no need to keep this block either if it’s not needed, let’s get rid of it

**Review by jstoiko — APPROVED** [review]

**Pull request merged**